src/app/components/server-charts/server-charts.component.ts

```
@@ -0,0 +1,791 @@
+  }
+
+  public generateMultiBarChart(chartTemplate: ChartFromBenchmarkTemplate) {
+
+    const chartConf = chartTemplate.options[chartTemplate.selectedOption];
+    const labelsField = chartConf.labelsField;
+    const scaleField = chartConf.scaleField;
+    const dataSet = this.benchmarksByCategory?.find(x => x.benchmark_id === chartConf.benchmark_id);
+
+    let chartData: any;
+
+    if(dataSet && dataSet.benchmarks?.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)